integration-tests/contract-with-bad-code.spec.ts

```diff
@@ -3,13 +3,13 @@ import { badCode } from "./data/badCode";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract failing with bad code using: ${rpc}`, () => {
+  describe(`Test contract origination fail with bad code through contract api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('fails with a 400 RPC response', async (done) => {
+    it('Verify that contract.originate for a contract with 400 in code will return 400 RPC in response', async (done) => {
```

Comment:
We could use the same definition as the one in the corresponding wallet test:
`Verify that contract.originate for a contract with a missing instruction (FAILWITH_TYPO) in code will return 400 RPC in response`

Reply:
changed to 'Verify that contract.originate for a contract with a missing instruction (FAILWITH_TYPO) in code will return 400 RPC in response'